package.json

```diff
@@ -34,8 +34,8 @@
     "localforage": "1.7.3",
     "localforage-observable": "2.0.0",
     "mark.js": "8.11.1",
-    "marked": "github:mattermost/marked#8214e10918264fb88cec474196023949f7cb4b30",
-    "mattermost-redux": "github:mattermost/mattermost-redux#66314024377668f3782e4c3a2df262ededa3a840",
+    "marked": "github:mattermost/marked#3a8d6ffc804de8ecd6b749ef8ebc9a4a94067b80",
+    "mattermost-redux": "github:mattermost/mattermost-redux#dbb9757be04e63083e0dcd1f8efe1004bd643000",
```

Comment:
Unfortunately, something breaks by the mattermost-redux but should be fixed by recent https://github.com/mattermost/mattermost-redux/commit/50fd53a06a6d496645c6da8d55a16ec54ee00550